src/daft-connect/src/translation/logical_plan/local_relation.rs

```diff
@@ -0,0 +1,209 @@
+use std::{collections::HashMap, io::Cursor, sync::Arc};
+
+use arrow2::io::ipc::{
+    read::{StreamMetadata, StreamReader, StreamState},
+    IpcField, IpcSchema,
+};
+use arrow_format::ipc::MetadataVersion;
```

Comment:
it is an enum (not just a value of `4`) how else woudl we d o it?



```rust
                    pub enum MetadataVersion {
                        V1 = 0,
                        V2 = 1,
                        V3 = 2,
                        V4 = 3,
                        V5 = 4,
                    }
```

https://github.com/Eventual-Inc/Daft/blob/97c8023c63973abc7c250a1e606df170ce7b9292/src/arrow2/src/io/ipc/read/stream.rs#L20-L29

Reply:
we can do this:
`version: 4.try_into().unwrap(),`

Reply:
we could also change `arrow2` to reexport `MetadataVersion`

Reply:
I think that might actually be better. 

https://github.com/universalmind303/Daft/blob/f636d748ee1e92b432139af7e82b03666a66ca24/src/arrow2/src/io/ipc/read/mod.rs#L45

we can just change that to `pub` instead of `pub(crate)`